.github/workflows/docs.yml

```diff
@@ -110,6 +110,7 @@ jobs:
         if: steps.cache-examples.outputs.cache-hit != 'true'
         working-directory: modflow6
         run: |
+          sed -i 's/meson==1.3.0/meson>=1.7.1/g' ../modflow6-examples/etc/requirements.pip.txt
```

Comment:
maybe this substitution could replace the line regardless of what comes after meson? in case we change it on the examples side.

Reply:
modified so more general